frontend/src/components/PlayerOverview.tsx

```diff
@@ -19,7 +17,7 @@ export default function PlayerOverview(props: PlayerOverviewProps){
 
     function handleSubmit(e:FormEvent<HTMLFormElement>) {
         e.preventDefault()
-        props.backendService.createNewPlayer(playerName)
+        props.createNewPlayer(playerName)
             .then(() => props.getAllPlayers())
```

Comment:
Brauchst du diese Zeile, um die Player neuzuladen? 
Ich glaub das machst du noch in der createNewPlayer Methode im Custom Hook